backend/src/image/ImageService.ts

```diff
@@ -0,0 +1,41 @@
+import fs, { PathLike } from 'fs';
+import path from 'path';
+import _ from 'lodash';
+import { Files } from 'formidable';
+import config from 'config';
+import { Service } from 'typedi';
+import BlogError from '@src/common/error/BlogError';
+import ImageRepository from '@src/image/ImageRepository';
+import { ImageDoc } from '@src/image/Image';
+import { UploadImageParamDto } from './ImageDto';
+import { BlogErrorCode } from '../common/error/BlogErrorCode';
+
+@Service()
+export default class ImageService {
+  public constructor(private readonly _imageRepository: ImageRepository) {
+  }
+
+  public uploadImage = (paramDto: UploadImageParamDto) => {
+    const { files } = paramDto;
+    ImageService.moveImage(files);
+    this.saveImage(files);
+  };
+
+  private static moveImage = (files: Files): void => {
+    const newPath: PathLike = config.get('path.image');
+    _.keys(files).forEach((fileName) => {
+      const { path: filePath } = files[fileName];
+      const newFilePath = path.resolve(newPath.toString(), fileName);
+      try {
+        fs.renameSync(filePath, newFilePath);
+      } catch (err) {
+        throw new BlogError(BlogErrorCode.FILE_CANNOT_BE_MOVED, [filePath, newFilePath]);
+      }
+    });
+  };
+
+  private saveImage = (files: Files): void => {
```

Comment:
`static` keyword can be added here because it has no dependency with other modules.